maintainers/maintainer-list.nix

```
@@ -3079,6 +3079,12 @@
     githubId = 1689801;
     name = "Mikhail Chekan";
   };
+  chen = {
+    email = "i@cuichen.cc";
+    github = "cu1ch3n";
+    githubId = 80438676;
+    name = "Chen Cui";
+  };
```

```suggestion
  cu1ch3n = {
    email = "i@cuichen.cc";
    github = "cu1ch3n";
    githubId = 80438676;
    name = "Chen Cui";
  };
```
You should use your github username `cu1ch3n` as the handle.